File: src/CorporateMessengerWithAgile/Application/Dto/Summary/MessageSummaryDto.cs
namespace Application.Dto.Summary
{
    public class MessageSummaryDto : BaseDto
    {
        public string Content { get; set; } = null!;
        public Guid ChatId { get; set; }
        public Guid SenderId { get; set; }
        public bool IsEdited { get; set; }
    }
}

File: src/CorporateMessengerWithAgile/ReactApp/reactapp.client/src/models/entity/MessageSummaryDto.ts
// src/models/entity/MessageSummaryDto.ts
import { z } from 'zod';
import { BaseDtoSchema } from './BaseDto';
import { GuidSchema } from '../Guid';

export const MessageSummaryDtoSchema = BaseDtoSchema.extend({
    content: z.string(),
    chatId: GuidSchema,
    senderId: GuidSchema,
    isEdited: z.boolean()
});

export type MessageSummaryDto = z.infer<typeof MessageSummaryDtoSchema>;